### DISCUSSION TITLE
Black screen after installing ExplorerPatcher on Windows build 22621.2361

### DISCUSSION THREAD

**Discussion by Kinnyr90** [valinet/ExplorerPatcher#2219] [Q&A]
I really need some help.

I have windows 11 22h2 with the latest september 26th Moment 4 update. When I install explorer patcher I believe it's the latest version but I'm not sure. Well when I install it and then restart my computer I get a black Screen. I can use the task manager by hitting control alt  delete. But File explorer is not there. all I have is a black screen. So at that point. all I can do is is sign out get back to the password box for my windows password and press and hold the shift key and restart windows so I can get to the blue troubleshooting screen. And from there is a section to uninstall the latest update which will be in my case The moment 4 update released nof the 26th of this month. So once I uninstall that I'm able to get back into windows. The latest build for windows 11 that I have is. 22621.2361. So. I guess I'm wondering if the latest version of Explorer patcher is compatable with this version along with the moment 4 update installed? It didn't work for me but that could be that I didn't have the latest version of explorer patcher installed.. Or do I have to wait for another version of the software to come out?

**Comment by Amrsatrio** [comment]
You surely do not have the latest version of ExplorerPatcher. Otherwise there would be an infinite number of new issues related to black screen on 226x1.2361.

**Comment by wookietv** [comment]
I had the same issue too when I started my computer this morning. If you run task manager > Run New Task > explorer.exe three times it will "crash" explorer patcher and disable it. Then you can update it to the current version which looks to have fixed it.

**Comment by Kinnyr90** [comment]
So If I have the latest version of Explorer Patcher it will work with Windows 11 with the moment 4 update??  Because it is Obviously an issue. I had past tense Explorer patcher on this computer with the Moment 4 update running on windows build 22621.2361 I would download the Moment 4 update and everything would be fine. Once I went and installed explorer patcher. after I restarted my computer. I got nothing but a black screen. so on the black screen I could hit control alst delete and get into task manager. But I was limited to what I could do because I didn't have file explorer anymore because explorer patcher crashed it. So I had to log out of windows. Get back to the password screen where you use your windows password to log into winbdows. and press and hold the shift key and restart the computer. And this would reboot the computer into the blue troubleshooting screen. from there. There is a section to uninstall the latest winbdows update anbd I did that. So once I did that It uninstalled the latest update which in my case was the moment 4 update released September 26th of this month. and then once I restarted my computer. I was able to get back into windows 11 And then If I wanted the moment 4 update I of course had to re download it, but first I had to uninstall the Explorer patcher software. So I did that and redownloaded the moment 4 update and everything was fine again. So where would I get the latest version can you send me the Link?
Thank you!
    On Saturday, September 30, 2023 at 03:26:41 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


You surely do not have the latest version of ExplorerPatcher.

**Reply by Amrsatrio** [reply]
> So If I have the latest version of Explorer Patcher it will work with Windows 11 with the moment 4 update??

Yes. I've tested on 22621.2361 and 22631.2361. There are some new quirks, but workarounds are available. Such as Win11 command bar must be used if you want Win11 context menu and EP kept resetting the Win11 taskbar settings to always combine.

> So where would I get the latest version can you send me the Link?

When you have a black screen:
1. Ctrl+Shift+Esc
2. Run new task
3. `rundll32 C:\Windows\dxgi.dll,ZZGUI`

And here in the EP settings:
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/8900273f-669a-4474-93ba-b6e34350b580)

You actually do not need to go to the recovery screen and stuff. We are really sorry for putting you into this situation.

**Comment by Kinnyr90** [comment]
Yeah. I didn't have to do that. Right after I installed explorer patcher with the Moment 4 update. I got the black screen. and as I said I had to go and uninstall the latest update from the blue troubleshooting screen and after I did that I was back in windows 11. But explorer patcher was still installed. But there was no more conflict because The moment 4 update was disabled. So If I wanted the Moment 4 update back on my pc I had to first uninstall explorer patcher. and that's what I did. So explorer patcher alone is not crashing windows 11 It's when I have both explorer patcher and the Moment 4 update together on my pc that's crashing File explorer and preventing me from getting into windows. Giving me a black screen and allowing me to hit Control alt delete to get to the task manager only to find out that file explorer is not showing. So could someone send me the link to the latest version of explorer patcher?
Thanks!
    On Saturday, September 30, 2023 at 08:49:02 AM EDT, wookietv ***@***.***> wrote:  
 
 


I had the same issue too when I started my computer this morning. If you run task manager > Run New Task > explorer.exe three times it will "crash" explorer patcher and disable it. Then you can update it to the current version which looks to have fixed it.

**Reply by Amrsatrio** [reply]
The latest version is always in the releases section: https://github.com/valinet/ExplorerPatcher/releases

**Comment by Kinnyr90** [comment]
Okay and I of course don't want the pre release I want the second one down the latest version right?
I just hope this works. Oh just to give you some more info on my situation.If I install a registry script to bring back the windowws classic taskbar.and I restart file explorer in task manager. Well windows is fine. But it still crashes file explorer and all it does is blink when I click of the file explorer yellow folder. That I have pinned to my task bar. And going into the classic shell start menu. Which works fine no issue. I still get the same crash. So in this case here I have to Go into regedit which I'm able to do and delete the HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Shell\Update\Packages. And I just have to delete this key here. UndockingDisabled=1 delete just this key restart the computer and the windows 11 taskbar is back but I want windows 10 taskbar. So now I don't see any other way that will not crash My computer except for Explorer patcher. Now you said explorer patcher was tested on my windows 11 build. But was it tested on a system with Moment 4 update installed? as well. Because that is where I'm having the issue. Explorer patcher will install to this Build as long as the moment 4 update is (not) installed.
    On Saturday, September 30, 2023 at 09:13:19 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


The latest version is always in the releases section: https://github.com/valinet/ExplorerPatcher/releases

**Reply by Amrsatrio** [reply]
Moment 4 is 22621.2361 and 22631.2361. Installing prerelease is generally okay, but there are times where it's buggy.

**Comment by Kinnyr90** [comment]
Okay... Yes your right!! So yes I will download the Latest release from the link you provided in your last email.
Okay I guess this closes this discussion if I have any firther issues I will get back to you. 
Thank you so much!
Have a great day!!God Bless!!
    On Saturday, September 30, 2023 at 09:55:10 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


Moment 4 is 22621.2361 and 22631.2361. Installing prerelease is generally okay, but there are times where it's buggy.

**Comment by Kinnyr90** [comment]
It worked. I have the latest release installed Thank you so much!! Once again.
    On Saturday, September 30, 2023 at 09:55:10 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


Moment 4 is 22621.2361 and 22631.2361. Installing prerelease is generally okay, but there are times where it's buggy.

**Reply by Amrsatrio** [reply]
Glad it worked! Remember to keep EP updated so that new Windows updates won't break your PC, okay?

**Comment by Kinnyr90** [comment]
Will do. Thanks again.
    On Saturday, September 30, 2023 at 11:00:38 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


Glad it worked! Remember to keep EP updated so that new Windows updates won't break your PC, okay?

**Comment by Kinnyr90** [comment]
Hi It's Steve Again.
Does the latest explorer patcher support 23h2 for windows 11?
    On Saturday, September 30, 2023 at 11:13:02 AM EDT, ***@***.*** ***@***.***> wrote:  
 
  Will do. Thanks again.
    On Saturday, September 30, 2023 at 11:00:38 AM EDT, Amr Satrio ***@***.***> wrote:  
 
 


Glad it worked! Remember to keep EP updated so that new Windows updates won't break your PC, okay?

**Comment by tfgmail** [comment]
Hi,

I have latest version ,but the screen goes black when i try to do remote installation using pdq deploy to our client machine

in pdq deploy it shows like the installation is success

The screen goes black after the installation then if i do force shutdown then it starts working normally and i can see that explorerpacther is working

but its not ideal to do a force restart for all machines manually

Could you please help with this?

**Reply by pyrates999** [reply]
This is already closed.  Please start a new discussion.
